PRS6001-7000/PRS6820MaryMag.xml

```diff
@@ -45,18 +45,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                                                   google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: person</change>
          <change who="PL" when="2019-03-27">removed occupation containing St, Saint, bibl, biblical or prophet, added keyword. See https://github.com/BetaMasaheft/Documentation/issues/1104</change>
+         <change when="2023-11-23" who="CH">updated names</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
             <person sex="2">
```

Comment:
wd Q63070

(and add bm PRS6820MaryMag in wikidata)

Reply:
I logged myself in to wikidata, but I cannot find a button to add the BetMas Identifier. Is it possible, that this record is protected and I cannot modify it?